packages/cli/cli/src/cli-context/upgrade-utils/getLatestVersionOfCli.ts

```diff
@@ -9,12 +9,7 @@ export async function getLatestVersionOfCli({
     cliEnvironment: CliEnvironment;
     includePreReleases?: boolean;
 }): Promise<string> {
-    // when running a non-published version of the CLI (e.g. in ETE tests),
-    // don't try to upgrade
-    if (cliEnvironment.packageVersion === "0.0.0") {
-        return cliEnvironment.packageVersion;
-    }
-    return latestVersion(cliEnvironment.packageName, {
+    return latestVersion("fern-api", {
```

Comment:
hardcoding `"fern-api"` because now when running `fern upgrade` with the dev-cli I get `Package `@fern-api/fern-api-dev` could not be found`